### ISSUE TITLE
Add Alertmanager firing alerts dashboard to custom mixin

### ISSUE THREAD

**Issue by ArthurSens** [prometheus-operator/kube-prometheus#1065]
**What is missing?**
Earlier today I've opened https://github.com/prometheus/alertmanager/issues/2529 asking for 2 dashboards related to Alertmanager, but now I think it would make more sense to implement the second idea here.

The idea is that kube-prometheus could add [Alertmanager grafana datasource plugin](https://grafana.com/grafana/plugins/camptocamp-prometheus-alertmanager-datasource/), by default or addon, and also include a dashboard that shows all firing alerts, split by severity.

**Why do we need it?**
While `critical` alerts are often routed to pagers and solved quickly, `warning` and `info` alerts may not be routed at all even though they provide useful information. This dashboard could provide extra guidance on daily monitoring activities or during incident responses.

**Comment by lilic** [comment]
Thanks, sounds like an interesting issue! I personally am not a fan in watching dashboards, only useful IMHO during an incident. How would that dasboard look like? Thanks!

cc @prometheus-operator/kube-prometheus-reviewers

**Comment by ArthurSens** [comment]
Using my use-case as an example, every PR on [gitpod-io/gitpod](https://github.com/gitpod-io/gitpod) creates a new dedicated namespace to that PR on our development cluster. It can happen that a bogus PR breaks some stuff on the cluster, or it might happen that a lot of PRs come up at the same time and cluster resources start to get limited.

We've just rolled out kube-prometheus to that cluster, but we don't see that cluster as critical yet. Every developer on the team has access to it and they can solve things on their own to keep the development going. We don't want to be paged nor have tickets created because of this cluster, but at the same time, the alerts could give context to developers that might want to fix this cluster quickly.

It doesn't need to be pretty, just dump alerts with common annotations. Here's a draft I've created this morning:
![image](https://user-images.githubusercontent.com/24193764/112997708-bb87af00-9143-11eb-8abb-60568d340547.png)
